### DISCUSSION TITLE
Spinner elements still appear on results screen even when map has no spinners

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#21309] [Q&A] [answered]
One thing I noticed about the results screen is that when there are no sliders, the Slider Tick and Slider End elements disappear, which makes sense. However, Spinner Bonus and Spinner Spin aren't treated in the same way and they will always appear on the results screen, even when the map doesn't actually have any spinners. Personally, it feels a bit inconsistent to me.

Map with sliders and spinners:
![image](https://user-images.githubusercontent.com/74088315/202834627-014395fd-dda5-4cbe-a455-f61c66c5bf6f.png)

Map with spinners but no sliders (no Slider Tick or Slider End):
![image](https://user-images.githubusercontent.com/74088315/202834638-0056786c-43f9-425f-a07a-3609ef365981.png)

Map with sliders but no spinners (still has Spinner Bonus and Spinner Spin):
![image](https://user-images.githubusercontent.com/74088315/202834675-ca87ac42-f646-41be-836c-3f7853311904.png)

Map with no sliders and no spinners (still has Spinner Bonus and Spinner Spin):
![image](https://user-images.githubusercontent.com/74088315/202834707-7109360f-eeeb-4037-a721-5bc160856e96.png)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/21350
